spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/src/test/java/org/springframework/cloud/dataflow/tasklauncher/TaskLauncherFunctionApplicationTests.java

```diff
@@ -76,8 +71,7 @@ public void taskPlatformAtCapacity() {
 		LaunchRequest launchRequest = new LaunchRequest();
 		launchRequest.setTaskName("someTask");
 		setCurrentExecutionState(3);
-		Optional<LaunchResponse> taskId = taskLauncherFunction.apply(launchRequest);
-		assertThat(taskId.isPresent()).isFalse();
+		assertThatThrownBy(() -> taskLauncherFunction.accept(launchRequest)).isInstanceOf(SystemAtMaxCapacityException.class);
```

Comment:
The `assertThatExceptionOfType(SystemAtMaxCapacityException.class).isThrownBy(() -> ...` is a handy variant for this sort of assertion.

Reply:
+1